File: runtime/src/de/fatox/meta/api/dao/MetaShaderData.java
package de.fatox.meta.api.dao;

import com.google.gson.annotations.Expose;

/**
 * Created by Frotty on 18.07.2016.
 */
public class MetaShaderData {
    @Expose
    private GLShaderData glShaderData;
    @Expose
    private MetaUniformData[] uniforms;

}

File: runtime/src/de/fatox/meta/api/dao/MetaShaderData.kt
package de.fatox.meta.api.dao

import com.badlogic.gdx.utils.Array

/**
 * Created by Frotty on 18.07.2016.
 */
data class MetaShaderData(var glShaderData: GLShaderData = GLShaderData(),
                          var uniforms: Array<MetaUniformData>) {

}
